scripts/utilities/browse_hdf5.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+import lz4.frame
+import pickle
+import hist
+from utilities import boostHistHelpers as hh, logging
+#import wremnants
+import hdf5plugin
+import h5py
+import narf
+from narf import ioutils
+import ROOT
+
+logger = logging.child_logger(__name__)
+
+if __name__ == "__main__":    
+
+     parser = argparse.ArgumentParser()
+     parser.add_argument("inputfile", type=str, nargs=1, help="Input file")
+     parser.add_argument("-w", "--what", type=str, default="all",
+                         choices=["all", "procs", "hists", "axes"],
+                         help="What to print")
+     parser.add_argument("-p", "--process", type=str, default=None,
```

Comment:
Given that the different modes do quite different things, some of the options aren't self-consistent. Using subparsers could be a solution (then add different options for the unique modes, it's done in the pullsAndImpacts script for example. Another option would be to have different scripts that access common functions in input_tools

Reply:
It's a good idea to have a subparser, but I won't be able to do it soon. In case somebody wants to do it sooner feel free to merge and edit it in a another PR, otherwise I'll do it as soon as I can.